scripts/token-mold.js

```diff
@@ -1317,11 +1313,12 @@ class TokenMoldForm extends FormApplication {
     // also populate with some calculated data for dnd5e, that is not in the template.json
     if (/dnd5e|sw5e/.exec(game.data.system.id) !== null) {
       let sortFun = function (a, b) {
-        if (a.attribute > b.attribute) return 1;
-        else if (a.attribute < b.attribute) return -1;
+        if (a.attribute > b.attribute) { return 1; }
+        else if (a.attribute < b.attribute) { return -1; }
         return 0;
       };
-      for (let skill of Object.keys(game.system.model.Actor["npc"].skills)) {
+      const npc = shells.find(o => o.type === 'npc');
+      for (let skill of Object.keys(npc.toObject().system.skills)) {
```

Comment:
same here:  do we need toObject()?  Kind of comes back to what is the data type of `shells`?

Reply:
The `toObject()` was copied from the example code here: https://foundryvtt.com/article/active-effects/

I assume it's required (or optimal).